test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/ListPackageTests.cs

```diff
@@ -399,6 +399,52 @@ public async Task GetReportDataAsync_WhenReportTypeIsVulnerable_ShouldUseAuditSo
             Assert.Equal("https://test/", result.Item2.Projects[0].TargetFrameworkPackages[0].TopLevelPackages.First().Vulnerabilities.First().AdvisoryUrl.ToString());
         }
 
+        [Fact]
+        public async Task GetReportDataAsync_WhenSolutionFileOutsideCWD_ShouldList()
+        {
+            // Arrange
+            using var mockServer = SetupMockServer();
+            var auditSource = new PackageSource(mockServer.Uri + "v3/index.json") { AllowInsecureConnections = true };
+
+            var mockRenderer = new Mock<IReportRenderer>();
+            var mockLogger = new Mock<ILogger>();
+
+            using var pathContext = new SimpleTestPathContext();
+            var solution = SetupTestSolution(pathContext);
+            SetupAssetsAndProps(Path.Combine(solution.Projects[0].SolutionRoot, solution.Projects[0].ProjectName).ToString());
+
+            var listPackageArgs = new ListPackageArgs(
```

Comment:
How are we specifying the CWD?

Reply:
Talked offline. We'll improve the test name, this is really just a test for list package on a solution, which is all we need.